hack/cleanup-ag.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires: jq, kubectl, cmk (get https://github.com/apache/cloudstack-cloudmonkey/releases/tag/6.4.0-rc1 or later)
+#
+# About: this tool helps to remove CloudStack affinity groups from CAPC
+# management cluster which are not assigned to any instances.
+#
+# Usage and help:
+# chmod +x cleanup-ag.sh
+# ./cleanup-ag.sh -h
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export DRY_RUN=false
+export VERBOSE=false
+export NAMESPACE=default
+export KUBECONFIG=$HOME/.kube/config
```

Comment:
Are _all_ of these consumed by child processes? If not, we can drop the `export`.

Reply:
Thanks, checked all and fixed